api/envoy/config/resource_monitor/unused_heap/v2alpha/BUILD

```diff
@@ -0,0 +1,11 @@
+# DO NOT EDIT. This file is generated by tools/proto_sync.py.
```

Comment:
add to https://github.com/envoyproxy/envoy/blob/master/api/docs/BUILD#L12 and run proto_format again to generate v3 protos.